core/assets/bundles/bundle_de.properties

```diff
@@ -929,19 +935,20 @@ block.scorch.name = Scatter
 block.scatter.name = Luftgeschütz
 block.hail.name = Streuer
 block.lancer.name = Lanzer
-block.conveyor.name = Förderband
-block.titanium-conveyor.name = Titan-Förderband
-block.armored-conveyor.name = Gepanzertes-Förderband
-block.armored-conveyor.description = Moves items at the same speed as titanium conveyors, but possesses more armor. Does not accept inputs from the sides from anything but other conveyors.
+block.conveyor.name = Transportband
+block.titanium-conveyor.name = Titan-Transportband
+block.armored-conveyor.name = Gepanzertes-Transportband
+block.armored-conveyor.description = Bewegt Material mit der gleichen Geschwindigkeit wie ein Titan-Transportband, hat aber eine bessere Panzerung. Nur andere Transportbänder können Material von der Seite anliefern.
 block.junction.name = Kreuzung
 block.router.name = Verteiler
 block.distributor.name = Großer Verteiler
-block.sorter.name = Sortierer
-block.inverted-sorter.name = Inverted Sorter
+block.sorter.name = Seiten-Sortierer
+block.inverted-sorter.name = Geradeaus-Sortierer
```

Comment:
Um es dem englischem gleich zu lassen, wäre es besser, sorter als "Sortierer" zu lassen. Außerdem koennte es problematisch mit der interpretation von "Geradeaus-Sortierer" geben, dah der inverted sorter ja dinge, die sortiert werden, nach aussen sortiert.
Waere dann inverted sorter nicht Seiten-Sortierer?

Reply:
Hier habe ich "Zur Seite sortieren" nach "Seiten-Sortierer" verkürzt.
Grundsätzlich fände ich es gut, wenn der Blockname gleich beschreibt, was er tut.
Sortieren als Wort beschreibt nicht, dass etwas zur Seite ausgegeben wird.
(Beim Geradeaus-Sortierer könnte man statt Geradeaus auch das Wort Vorwärts verwenden)


Ich habe mir auch den Namen Geradeaus-Priorisierer überlegt. Es wäre dann aber zu weit vom ursprünglichen Namen entfernt.

(Ich hätte am liebsten, wenn man den Block einstellen kann und ihn nicht so auswählen muss.  Das wäre aber was für ein separates Feature Request)

Reply:
Noch Fragen hierzu?
(Ich persönlich bin auch mit dem Englischen Sorter/Inverted Sorter als Text unzufrieden. Ich hätte hier lieber Side-Sorter und Front-Sorter, oder sogar das Wort Priorisierer. Z.B. Front-Prioritizer and Side-Prioritizer)